src/style.css

```
@@ -0,0 +1,108 @@
+* {
+  box-sizing: border-box;
+}
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir.ttc);
+}
+@font-face {
+  font-family: Avenir;
+  font-weight: 400;
+  src: url(fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
```

Add blank lines between rules
```suggestion
* {
  box-sizing: border-box;
}

@font-face {
  font-family: Avenir;
  font-weight: 300;
  src: url(fonts/Avenir.ttc);
}

@font-face {
  font-family: Avenir;
  font-weight: 400;
  src: url(fonts/Avenir-Book.ttf);
}

@font-face {
  font-family: Avenir;
  font-weight: 900;
  src: url(fonts/Avenir-Heavy.ttf);
}
```